lib/lob/resources/resource_base.rb

```diff
@@ -92,7 +92,7 @@ def build_nested_query(value, prefix = nil)
       end
 
       def base_url
-        "#{config[:protocol]}://#{config[:api_key]}:@#{config[:api_host]}/v1"
```

Comment:
Is there a reason why we allowed protocol and host, and is it OK to remove this functionality?